oggm/core/gcm_climate.py

```diff
@@ -58,30 +59,36 @@ def process_gcm_data(gdir, filesuffix='', prcp=None, temp=None,
     if (np.abs(temp['lon']) > 180) or (np.abs(prcp['lon']) > 180):
         raise ValueError('We expect the longitude coordinates to be within '
                          '[-180, 180].')
+    
 
-    # from normal years to hydrological years
-    sm = cfg.PARAMS['hydro_month_' + gdir.hemisphere]
-    prcp = prcp[sm-1:sm-13].load()
-    temp = temp[sm-1:sm-13].load()
-
-    # compute monthly anomalies
-    # of temp
+    ##################################################################################
+    # ISSSUE: in glacier model intercomparison is it useful to do that ?    
+    # From normal years to hydrological years
+    #sm = cfg.PARAMS['hydro_month_' + gdir.hemisphere]
+    #prcp = prcp[sm-1:sm-13].load()
+    #temp = temp[sm-1:sm-13].load()
```

Comment:
Yeah, we need to keep that. This is how OGGM works. When we output the data we can do something else, maybe we'll miss the last year or so

Reply:
Ok, and it also makes more sense. However sometimes experiment requirements are not the same